examples/add_partitions.py

```diff
@@ -0,0 +1,22 @@
+from hive_metastore_client.builders import PartitionBuilder
+from hive_metastore_client.hive_mestastore_client import HiveMetastoreClient
+
+HIVE_HOST = "<ADD_HIVE_HOST_HERE>"
+HIVE_PORT = 9083
+
+DATABASE_NAME = "database_name"
+TABLE_NAME = "table_name"
+
+# partitions should be provided as list
+partition_list = [
+    PartitionBuilder(
+        values=["2020", "12", "13"], db_name=DATABASE_NAME, table_name=TABLE_NAME,
```

Comment:
Even though it may be pretty obvious, I think you could add a comment mentioning that the table was created with the partition keys year, month, and day. And the user must follow this order.